src/utils/useHotspot.ts

```diff
@@ -111,14 +119,22 @@ const useHotspot = () => {
     return parsedStr
   }
 
-  const connectAndConfigHotspot = async (hotspotDevice: Device) => {
+  const handleConnectStatus = (status: HotspotConnectStatus) => {
+    if (status !== 'success') {
+      Logger.error(new Error(`Hotspot connect failed: ${status}`))
+    }
+    return status
+  }
+
+  const connectAndConfigHotspot = async (
+    hotspotDevice: Device,
+  ): Promise<HotspotConnectStatus> => {
     let connectedDevice = hotspotDevice
     const connected = await hotspotDevice.isConnected()
     if (!connected) {
       const device = await connect(hotspotDevice)
       if (!device) {
-        Logger.error(new Error('Hotspot connect failed: no device'))
```

Comment:
Can we leave the logs in, just so we can tell when things are going wrong and where for people reporting issues

Reply:
handleConnectStatus will log